rules/no-multiple-start-events.js

```diff
@@ -0,0 +1,38 @@
+const { is } = require('bpmnlint-utils');
+
+const { ERROR_TYPES } = require('./utils/element');
+
+const { reportErrors } = require('./utils/reporter');
+
+module.exports = function() {
+  function check(node, reporter) {
+    if (!is(node, 'bpmn:Process')) {
+      return;
+    }
+
+    const flowElements = node.get('flowElements') || [];
+
+    const blankStartEvents = flowElements
+      .filter(e => is(e, 'bpmn:StartEvent'))
+      .filter(e => !e.eventDefinitions);
```

Comment:
`eventDefinitions` can be an empty array, so I'd rather suggest a safe option:
```suggestion
      .filter(e => !e.get('eventDefinitions').length);
```

Reply:
(this is because `!([]) === false`)

Reply:
This makes senseg good catch! I integrated the suggestion with [0905328](https://github.com/camunda/bpmnlint-plugin-camunda-compat/pull/104/commits/090532882c04c099f1f1d897490b240f80379ddf)